Options.cs

```diff
@@ -46,5 +60,15 @@ public string Get(string key)
         {
             return GetType().GetProperty(key, BindingFlags.Public | BindingFlags.Instance).GetValue(this) as string;
         }
+
+        public int GetPreferredDumpSpeedForMediaType(MediaType? type)
+        {
+            switch (type)
+            {
+                case MediaType.CD: return preferredDumpSpeedCD;
+                case MediaType.DVD: return preferredDumpSpeedDVD;
+                default: return 1; //TODO: think what we want here
```

Comment:
The "reasonable default" we use right now is 8, I think that would be decent for this as well.